ngraph/python/tests/test_onnx/utils/model_importer.py

```diff
@@ -123,11 +127,16 @@ def _execute_pb_data(
             if(len(inputs) == 0):
                 continue
             outputs = list(prepared_model.run(inputs))
+            if post_processing is not None:
+                outputs = post_processing(outputs)
             cls.assert_similar_outputs(ref_outputs, outputs, result_rtol, result_atol)
             executed_tests = executed_tests + 1
         return executed_tests
 
-    def _add_model_execution_test(self, model_test: OnnxTestCase) -> None:
+    def _add_model_execution_test(
```

Comment:
Can we avoid adding `post_processing` as a param to multiple functions?

Reply:
I've created `ExtOnnxTestCase` which extends `OnnxTestCase` from ONNX library so one adding was removed